# <TODO>

## Rule Details

TODO

Examples of **incorrect** code for this rule:

```js
html`TODO`;
```

Examples of **correct** code for this rule:

```js
html`TODO`;
```

## When Not To Use It

TODO

## Further Reading

[TODO](TODO)
